integrations/extensions/starter-kits/language-model-watsonx/README.md

```diff
@@ -2,23 +2,26 @@
 
 watsonx is our enterprise-ready AI and data platform designed to multiply the impact of AI across your business. It provides an API for interacting with generative language models. You can use this starter kit to connect IBM Watson Assistant to the watsonx API.
 
-This starter kit does not provide examples of anything that you can _do_ with this extension. It can be used for specific use cases as the generative language model component.
+## Example use cases with the watsonx custom extension
+
+- [Complex informational tasks: Use watsonx to generate natural language answers for specific topics with only a small amount of relevant text](../language-model-complex-informational-tasks/README.md)
```

Comment:
The watsonx kit for this one is not merged yet.  Should we wait to merge this change until that one is merged too?

Reply:
Yes, it makes sense to wait.